MSSQL/OMOPLoader.sql

```diff
@@ -23,7 +23,7 @@
 ----------------------------------------------------------------------------------------------------------------------------------------
 
 -- Change to your omop database
-use i2b2stub;
+use AllOfUs_Mart;

```

Comment:
Please take this out, nothing project specific should be in the transform. Thanks!

Reply:
Agreed. Reverted in latest commit.